### PR TITLE
feat(project-access): get CAP environment config

### PR DESCRIPTION
**Pull request by voicis**
### Ticket: 
#1015 

### Description
Introduce new utility function for getting a CAP project environment.

### CODE REVIEW

**Review by Klaus-Keller — APPROVED** [review]
- code change looks good and easy to understand
- test coverage is great
- build locally and tested using
    ```Shell
    packages/project-access/dist$ node -e 'require("./index").getCapEnvironment("PATH_TO_CAP").then(env=>console.log(env))'
    ````
    and could not find any issue.

Approved from my side.

File: packages/project-access/src/project/cap.ts [hunk 1]
```diff
@@ -96,3 +95,15 @@ export async function getCapModelAndServices(
...
 }
+
+/**
+ * Get CAP CDS project environment config for project root.
+ *
+ * @param capProjectPath - project root of a CAP project
+ * @returns - environment config for CAP project
+ */
+export async function getCapEnvironment(capProjectPath: string): Promise<CdsEnvironment> {
+    const cds = await loadModuleFromProject<any>(capProjectPath, '@sap/cds');
```
Comment (tobiasqueck) [inline packages/project-access/src/project/cap.ts:106]: `unknown` would be the better type here.

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
small comment regarding type usage

File: packages/project-access/src/project/cap.ts (hunk 1 above)
Comment (voicis) [reply packages/project-access/src/project/cap.ts:106]: If I use `unknown` then we get compiler error in the next line. We would need a specific type instead, I suppose we could do that as well.

Comment (tobiasqueck) [reply packages/project-access/src/project/cap.ts:106]: Do you know the type, then yes, please. Based on the next line, I assumed you don't know exactly what you get.

**Review by tobiasqueck — APPROVED** [review]
* use case and corresponding changeset is clear
* code adheres to coding standards and is properly tested
* did not manually test it

**Review by Klaus-Keller — APPROVED** [review]
I re-reviewed the newly added type, looks good to me.
Again approved from my side.

**Pull request merged**